core/drivers/crypto/stm32/stm32_saes.c

```diff
@@ -1435,7 +1436,7 @@ static TEE_Result stm32_saes_probe(const void *fdt, int node,
 	if (res)
```

Comment:
Nice catch, I think we can remove the assert.
It was to protect if we try to probe 2 instances of SAES but OP-TEE does not support several AES accelerator.
So It will fail during the register in crypto framework

Reply:
addressed